README.md

```diff
@@ -241,6 +241,33 @@ If your repo uses `EMOJI-LOG` then you can add any of the following badges to yo
 
 <br>
 
+(graphic needed here `@TODO`)
+
+### Alfred Snippets
+
+[Alfred](https://www.alfredapp.com/) [PowerPack](https://www.alfredapp.com/powerpack/) users can use the Snippets feature to quickly call Emoji-Log, or use the text expand feature for even quicker creation.
+
+To setup:
+
+1. Have Alfred 3 with PowerPack installed
+2. For auto expansion, in _Alfred Settings » Features » Snippets_ ensure the "Automatically expand snippets by Keyword" box is checked
+3. Download & open [`Emoji-Log.alfredsnippets`](Emoji-Log.alfredsnippets), deselecting "Strip snippets of 'auto expand' flag" when prompted
+
+This will give the following text expander keywords for the Emoji-Log:
+
+| Keyword | Snippet |
+| --- | --- |
+| `;new` | 📦 NEW: |
+| `;improve` | 👌 IMPROVE: |
+| `;fix` | 🐛 FIX: |
+| `;release` | 🚀 RELEASE: |
+| `;doc` | 📖 DOC: |
+| `;test` | ✅ TEST: |
+
+To edit the `;` prefix to your preferred expansion flag, double click right click the Emoji-Log Collection in _Alfred Settings » Features » Snippets_.
```

Comment:
I use the same exact setup for TextExpander but with a `g` prefix.

| Keyword | Snippet |
| --- | --- |
| `;gnew` | 📦 NEW: |
| `;gimp` | 👌 IMPROVE: |
| `;gfix` | 🐛 FIX: |
| `;grlz` | 🚀 RELEASE: |
| `;gdoc` | 📖 DOC: |
| `;gtst` | ✅ TEST: |


Can you consider to change the snippets to reflect that? It's easier to remember and namespaced snippets since `g` stands for `git`.

Reply:
Yes great point! I missed that that was the naming convention in the shell functions. Fixed in https://github.com/ahmadawais/Emoji-Log/pull/14/commits/25e92557cccdf4ba27c8ed829c870c4f58f56efd